From 1539117c97008732958fb54b2469e014f549ea89 Mon Sep 17 00:00:00 2001
From: Serguei Katkov <serguei.i.katkov@intel.com>
Date: Tue, 3 Dec 2013 13:45:20 +0700
Subject: Dalvik: Clean-up emulator re-enabling

By mistake Init.cpp contains ifndef ARCH_IA32 instead of ifdef

This is fixed

Category: device-enablement
Domain: AOSP-Dalvik-Runtime
Origin: internal
Upstream-Candidate: no, needs rework

BZ: APDEV-590

Change-Id: I9dc1350bb80e8bf7b74a3d3a6b2fcaaa71778437
Signed-off-by: Serguei Katkov <serguei.i.katkov@intel.com>
---
 vm/Init.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vm/Init.cpp b/vm/Init.cpp
index 8924572..f456feb 100644
--- a/vm/Init.cpp
+++ b/vm/Init.cpp
@@ -1466,7 +1466,7 @@ int processOptions(int argc, const char* const argv[], bool ignoreUnrecognized)
             gDvmJit.printBinary = true;
         } else if (strncmp(argv[i], "-Xjitverbose", 12) == 0) {
             gDvmJit.printMe = true;
-#ifndef ARCH_IA32
+#ifdef ARCH_IA32
         } else if (strncmp(argv[i], "-Xjitbackendstring:", 19) == 0) {
             char *ptr = strchr (argv[i], ':');
 
-- 
1.7.4.1

